frontend/components/channel-sidebar/channel_list_item.jsx

```diff
@@ -14,10 +14,8 @@ class ChannelListItem extends React.Component {
   }
 
   handleSelection(){
-    const channel = this.props.channel;
-    const permissions = this.props.permissions;
-    if (!this.props.channel.private || permissions[channel.id])
-     { this.props.selectChannel(channel.id); }
+    const { selectChannel, channel } = this.props;
```

Comment:
this can fit on a single line without having to destructure it. its a preference thing at this point. im fine with it either way, just letting you know we don't ALWAYS have to destructure.